dom.bs

```diff
@@ -3594,6 +3625,60 @@ run these steps:
 <ol>
  <li><p>Assert: either <var>addedNodes</var> or <var>removedNodes</var> <a for=set>is not empty</a>.
 
+ <li><p>If either <var>options</var>'s' {{MutationObserverInit/elementLocalNameFilter}} or
+ {{MutationObserverInit/elementFilterByAttribute}} is present, then:
```

Comment:
Since this `<p>` isn't the only child of the `<li>`, it needs to be on its own line, indented one. This also goes for the following `<ol>`, which is not indented accurately.